test/expression/eval_test.exs

```diff
@@ -139,6 +139,19 @@ defmodule Expression.EvalTest do
     assert 4.0 == Eval.eval!(ast, %{})
   end
 
+  test "arithmetic with decimals" do
+    {:ok, ast, "", _, _, _} = Parser.parse("@(1.5 + 1.5)")
+    assert Decimal.new("3.0") == Eval.eval!(ast, %{})
+    {:ok, ast, "", _, _, _} = Parser.parse("@(1.5 + 2.5 * 3.5)")
+    assert Decimal.new("10.25") == Eval.eval!(ast, %{})
+    {:ok, ast, "", _, _, _} = Parser.parse("@((1.5 + 2.5) * 3.5)")
+    assert Decimal.new("14.00") == Eval.eval!(ast, %{})
+    {:ok, ast, "", _, _, _} = Parser.parse("@((1.5 + 2.5) / 3.5)")
+    assert Decimal.new("1.14286") == Decimal.round(Eval.eval!(ast, %{}), 5)
```

Comment:
the rounding here is because depending the mood of the cpu it'll print a decimal with different precisions